app/views/summary/UKSitesSummary.scala

```diff
@@ -91,28 +91,30 @@ object UKSitesSummary {
       )
   }
 
-  def getHeadingAndSummary(userAnswers: UserAnswers, isCheckAnswers: Boolean, subscription: RetrievedSubscription)
-                          (implicit messages: Messages): Option[(String, SummaryList)] = {
+  def getHeadingAndSummaryForCorrectReturn(userAnswers: UserAnswers, isCheckAnswers: Boolean, subscription: RetrievedSubscription)
+                                          (implicit messages: Messages): Option[(String, SummaryList)] = {
     val optSummaryList = (
       UserTypeCheck.isNewPacker(SdilReturn.apply(userAnswers), subscription),
-      UserTypeCheck.isNewImporter(SdilReturn.apply(userAnswers), subscription)
+      UserTypeCheck.isNewImporter(SdilReturn.apply(userAnswers), subscription),
+      subscription.productionSites.isEmpty,
+      subscription.warehouseSites.isEmpty
     ) match {
-      case (true, false) => Option(
+      case (true, false, true, _) => Option(
         SummaryListViewModel(
           Seq(
             getPackAtBusinessAddressRow(userAnswers, isCheckAnswers)
           )
         )
       )
-      case (true, true) => Option(
+      case (true, true, true, true) => Option(
         SummaryListViewModel(
           Seq(
             getPackAtBusinessAddressRow(userAnswers, isCheckAnswers),
             getAskSecondaryWarehouseRow(userAnswers, isCheckAnswers)
           )
         )
       )
-      case (false, true) => Option(
+      case (false, true, _, true) => Option(
```

Comment:
I'd probably move this about the case on line 109 stylistically. I think it would look neater.